### PR TITLE
[RUM-234] add workerUrl documentation

### PR DESCRIPTION
**Pull request by BenoitZugmeyer**
### What does this PR do?

Add documentation for the `workerUrl` Browser SDK initialization parameter. 
See related PR: https://github.com/DataDog/browser-sdk/pull/2354

### Motivation

The RUM Browser SDK is using a Web Worker. To facilitate distribution, this worker is built with a "blob" URL, thus requiring the CSP `worker-src blob:;` directive, which might be seen as a security risk.

With the new `workerUrl` initialization parameters, users will be able to self-host the worker JavaScript file to workaround CSP restrictions.

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/integrations/content_security_policy_logs.md [hunk 1]
```diff
@@ -182,6 +182,14 @@ If you are using Session Replay, make sure to allow workers with `blob:` URI sch
 worker-src blob:;
 ```
 
+Alternatively, starting from [version TODO][8], you can self-host the Datadog Browser SDK Worker JavaScript file and provide the `workerUrl` option to initialize the [RUM Browser SDK][8]. To do so, you can either:
+
+* download it from https://unpkg.com/@datadog/browser-worker , and store it alongside your Web application assets;
```
Comment (bcaudan) [inline content/en/integrations/content_security_policy_logs.md:187]:
```suggestion
* download it from https://unpkg.com/@datadog/browser-worker, and store it alongside your Web application assets;
```

File: content/en/integrations/content_security_policy_logs.md [hunk 2]
```diff
@@ -182,6 +182,14 @@ If you are using Session Replay, make sure to allow workers with `blob:` URI sch
 worker-src blob:;
 ```
 
+Alternatively, starting from [version TODO][8], you can self-host the Datadog Browser SDK Worker JavaScript file and provide the `workerUrl` option to initialize the [RUM Browser SDK][8]. To do so, you can either:
+
+* download it from https://unpkg.com/@datadog/browser-worker, and store it alongside your Web application assets;
+
+* or install the [`@datadog/browser-worker` NPM package][9] and use your build tool to include it in the built assets (see documentation for [Webpack 4][10], [Webpack 5][11], [Vite][12], [Rollup][13]).
+
+Keep in mind that you need to host it on the *same origin* as your Web application. It cannot be hosted on a separate domain (ex: no third party CDN host) or a different scheme. This is a browser restriction (see [MDN documentation][14]: "This script must obey the same-origin policy").
```
Comment (bcaudan) [inline content/en/integrations/content_security_policy_logs.md:191]: **💬 suggestion:** ‏we should maybe add a note saying something about the worker version matching the sdk version

Comment (BenoitZugmeyer) [reply content/en/integrations/content_security_policy_logs.md]: Done. I mentioned "*major* version", as CDN users won't maintain an exactly matching version and I wanted to keep it simple, but I'm open to suggestions

**Review by bcaudan — APPROVED** [review]
LGTM when the version will be defined

File: content/en/real_user_monitoring/browser/_index.md [hunk 3]
```diff
@@ -1537,9 +1537,14 @@ Telemetry data (such as errors and debug logs) about SDK execution is sent to Da
...
 `excludedActivityUrls`
 : Optional<br/>
-**Type:** List<br/>
+**Type**: List<br/>
 A list of request origins ignored when computing the page activity. See [How page activity is calculated][16].
 
+`workerUrl`
+: Optional<br/>
+**Type**: String<br/>
+URL pointing to the Datadog Browser SDK Worker JavaScript file. It can be either relative or absolute but is required to have the same origin as the Web application. See [Content Security Policy guidelines][22] for more information.
```
Comment (drichards-87) [inline content/en/real_user_monitoring/browser/_index.md:1546]:
```suggestion
URL pointing to the Datadog Browser SDK Worker JavaScript file. The URL can be relative or absolute, but is required to have the same origin as the web application. See [Content Security Policy guidelines][22] for more information.
```

File: content/en/integrations/content_security_policy_logs.md [hunk 4]
```diff
@@ -182,6 +182,18 @@ If you are using Session Replay, make sure to allow workers with `blob:` URI sch
 worker-src blob:;
 ```
 
+Alternatively, starting from [version 4.46.0][8], you can self-host the Datadog Browser SDK Worker JavaScript file and provide the `workerUrl` option to initialize the [RUM Browser SDK][8]. To do so, you can either:
```
Comment (drichards-87) [inline content/en/integrations/content_security_policy_logs.md:185]:
```suggestion
Alternatively, starting from [version 4.46.0][8], you can self-host the Datadog Browser SDK Worker JavaScript file and provide the `workerUrl` option to initialize the [RUM Browser SDK][8] by doing one of the following:
```

File: content/en/integrations/content_security_policy_logs.md [hunk 5]
```diff
@@ -182,6 +182,18 @@ If you are using Session Replay, make sure to allow workers with `blob:` URI sch
...
+
+* download it from https://unpkg.com/@datadog/browser-worker, and store it alongside your Web application assets;
+
+* or install the [`@datadog/browser-worker` NPM package][9] and use your build tool to include it in the built assets (see documentation for [Webpack 4][10], [Webpack 5][11], [Vite][12], [Rollup][13]).
+
+Requirements:
+
+* Make sure the Worker major version matches the Browser SDK version you are using.
+
+* Host this file on the *same origin* as your Web application. It cannot be hosted on a separate domain (ex: no third party CDN host) or a different scheme. This is a browser restriction (see [MDN documentation][14]: "This script must obey the same-origin policy").
```
Comment (drichards-87) [inline content/en/integrations/content_security_policy_logs.md:195]:
```suggestion
* Make sure the Worker major version matches the Browser SDK version you are using.
* Host the file on the same origin as your web application. Due to [browser restrictions][14], it cannot be hosted on a separate domain (for example, a third-party CDN host) or a different scheme.
```

**Review by drichards-87 — CHANGES_REQUESTED** [review]
Left you some minor feedback.

**Review by drichards-87 — APPROVED** [review]
Looks good. Let me know when this is ready to be merged.

**Comment by BenoitZugmeyer** [comment]
> Looks good. Let me know when this is ready to be merged.

@drichards-87 I fixed the SDK version, as this change wasn't released before. But it's all good now, you can merge it anytime.

**Pull request merged**